src/openrct2/scripting/Duktape.hpp

```diff
@@ -219,7 +219,7 @@ namespace OpenRCT2::Scripting
             auto it = _s2n.find(k);
             if (it == _s2n.end())
             {
-                return 0;
+                return (T)0;
```

Comment:
why not `return {0};`?

Reply:
That produces errors. I have changed it to `static_cast` instead.